snowflake-api/src/responses.rs

```diff
@@ -54,7 +56,7 @@ pub struct ExecErrorResponseData {
     pub pos: Option<i64>,
 
     // fixme: only valid for exec query response error? present in any exec query response?
-    pub query_id: String,
+    pub query_id: Option<String>,
```

Comment:
Made `query_id` optional as it could be null, for example when I cancel long query response has it as `queryId: null`